packages/@aws-cdk/aws-cloudtrail/lib/cloudtrail.ts

```diff
@@ -115,6 +115,17 @@ export interface TrailProps {
    * @default - if not supplied a bucket will be created with all the correct permisions
    */
   readonly bucket?: s3.IBucket;
+
+  /**
+   * Specifies whether the trail is applied to all accounts in an organization in AWS Organizations, or only for the current AWS account.
+   * The default is false, and cannot be true unless the call is made on behalf of an AWS account that is the management accountfor an organization in AWS Organizations.
+   * If the trail is not an organization trail and this is set to true, the trail will be created in all AWS accounts that belong to the organization.
+   * If the trail is an organization trail and this is set to false, the trail will remain in the current AWS account but be deleted from all member accounts in the organization.
+   *
```

Comment:
This documentation feels a little clunky to me. After the first read through I felt like I didn't quite get it so I went and looked up `CfnTrail` to try to get clarification. I see that it came from there but I think we can make this more clear.